marketplace_standard_app_api/main.py

```diff
@@ -108,29 +108,155 @@ async def heartbeat() -> HTMLResponse:
     return HTMLResponse(content="<html><body>OK</body></html>", status_code=200)
 
 
-@api.post(
+@api.get(
     "/datasets",
+    operation_id="listDatasets",
```

Comment:
It seems that this API doesn't support collections (i.e., sets of datasets), as opposed to conclusions of previous discussions with other developers. This could be useful to bundle together related datasets but I'm not sure if it's a must.

Reply:
I was not aware that collections were desired. I can update the API if you like, since they are natively supported by SWIFT and S3.

Reply:
This is implemented now.